### ISSUE TITLE
PR for nodejs?

### ISSUE THREAD

**Issue by treysis** [balena-io-modules/fetch#13]
Hey,

in https://github.com/nodejs/node/issues/6307 you said you'll craft a PR for happy eyeballs implementation on https://github.com/nodejs/node. So far, I haven't seen anything. Are you still planning on working on this?

Thx and keep up the great work!

**Comment by zwhitchcox** [comment]
Yes, it is still planned, but I'm not sure when I'll have time. It might be implemented in the node fetch api too, I'm not sure.
